build.sh

```diff
@@ -41,8 +41,11 @@ install_rpms() {
     local builddeps
     local frozendeps
 
-    # no frozen deps right now
+    # freeze grub2 due to https://github.com/coreos/coreos-assembler/issues/3148
     frozendeps=""
+    if [ "${arch}" != s390x ]; then
+        frozendeps=$(echo grub2{-common,-tools-extra}-2.06-52.fc36)
```

Comment:
this should be frozen only for the ppc64le platform right?

Reply:
I think it's better for us to stay consistent cross-architecture here personally.